### PR TITLE
Listing products Page

### PR DESCRIPTION
**Pull request by shivanshu2394**

### CODE REVIEW

File: TrainingProject/TrainingProject/Controllers/ProductController.cs [hunk 1]
```diff
@@ -20,6 +18,43 @@ public ProductController()
         }
 
         // GET: Product
+        [HttpGet]
+        public ActionResult Product_Listing()
```
Comment (salilsaini868) [inline TrainingProject/TrainingProject/Controllers/ProductController.cs]: Rename action method to "Listing"

File: TrainingProject/TrainingProject/Controllers/ProductController.cs [hunk 2]
```diff
@@ -28,7 +63,7 @@ public ActionResult Products_Index()
         [HttpPost]
         public ActionResult Products_Index(ProductModel prop)
         {
```
Comment (salilsaini868) [inline TrainingProject/TrainingProject/Controllers/ProductController.cs]: Rename method to  "InsertProduct"

File: TrainingProject/TrainingProject/Controllers/ProductController.cs (hunk 1 above)
Comment (shivanshu2394) [reply TrainingProject/TrainingProject/Controllers/ProductController.cs]: done

File: TrainingProject/TrainingProject/Controllers/ProductController.cs (hunk 2 above)
Comment (shivanshu2394) [reply TrainingProject/TrainingProject/Controllers/ProductController.cs]: done

**Pull request merged**